### PR TITLE
ProjectFileParser can be extracted in a user defined dir

### PR DESCRIPTION
**Pull request by tekuser**
Proposition for Issue https://github.com/Ullink/gradle-msbuild-plugin/issues/28

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,9 @@ Sample usage:
       // overrides project IntermediaryOutputPath
       intermediateDir = 'build/msbuild/obj"
       
+      // temporary directory where the plugin will extract the project file parser executable
```
Comment (gluck) [inline README.md]: I'd rather avoid making this configurable, it shouldn't be (many gradle plugins download&execute binaries, I've never seen on make the location configurable)

**Comment by gluck** [comment]
Do you know where the std location is for gradle-downloaded-binaries ? I've mostly seen ppl use HOME/.gradle, but not sure it's a standard

File: README.md (hunk 1 above)
Comment (tekuser) [inline README.md]: Yes that makes sense!

**Comment by tekuser** [comment]
Gradle download dependencies in the user's HOME/.gradle/  (there is a cache inside).
So gradle-msbuild-plugin.jar is downloaded somewhere in HOME/.gradle. But the plugin extract the ProjectFileParser.exe in the user's temp dir (defined by the env var %TEMP% or %TMP%).

So we can do the same thing as in the gradle-nuget-plugin:
https://github.com/Ullink/gradle-nuget-plugin/blob/master/src/main/groovy/com/ullink/BaseNuGet.groovy

 l28:       File nugetExe = new File(temporaryDir, NUGET_EXE)

**Pull request closed without merging**